packages/edit-site/src/components/sidebar-edit-mode/page-panels/reset-default-template.js

```diff
@@ -32,9 +25,6 @@ export default function ResetDefaultTemplate( { onClick } ) {
 				onClick={ async () => {
 					entity.edit( { template: '' }, { undoIgnore: true } );
 					onClick();
-					await setPage( {
```

Comment:
This is the kind of weird action calls that were needed because of the broken synchronization.